roles/etcd/templates/etcd.conf.j2

```diff
@@ -60,3 +60,10 @@ ETCD_PEER_CA_FILE={{ etcd_peer_ca_file }}
 ETCD_PEER_CERT_FILE={{ etcd_peer_cert_file }}
 ETCD_PEER_KEY_FILE={{ etcd_peer_key_file }}
 {% endif -%}
+
+#[logging]
+ETCD_DEBUG="{{ etcd_debug | default(false) | string }}"
+# examples for -log-package-levels etcdserver=WARNING,security=DEBUG
```

Comment:
Maybe remove this, just clutter -- someone should be in the `etcd` doc if they're touching this.